ovn-tester/ovn_utils.py

```diff
@@ -38,6 +46,9 @@
 DualStackIP = namedtuple('DualStackIP', ['ip4', 'plen4', 'ip6', 'plen6'])
 
 
+vlog.use_python_logger(max_level=vlog.INFO)
```

Comment:
There is one more thing we need to do.  Python-IDL is using select.select to simulate select.poll by default to avoid broken multithreading with select.poll implementation vs eventlet/gevent.  Unfortunately, that doesn't work if we're trying to open a lot of connections.  While running 500-node test, at some point (connecting the ovn-scale-325) the numerical value of a file descriptor surpasses 1024 and select() is unable to handle that throwing the exception:

```
2022-10-05 21:16:35,527 | ovsdbapp.backend.ovs_idl.vlog |INFO| ssl:192.16.1.74:6640: connecting...
2022-10-05 21:16:35,527 | ovsdbapp.backend.ovs_idl |ERROR| OVS database connection to Open_vSwitch failed with error: 'filedescriptor out of range in select()'. Verify that OVS and related services are available and that the relevant configuration options are correct.
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/ovsdbapp/backend/ovs_idl/__init__.py", line 109, in start_connection
    self.ovsdb_connection.start()
  File "/usr/local/lib/python3.10/site-packages/ovsdbapp/backend/ovs_idl/connection.py", line 83, in start
    idlutils.wait_for_change(self.idl, self.timeout)
  File "/usr/local/lib/python3.10/site-packages/ovsdbapp/backend/ovs_idl/idlutils.py", line 242, in wait_for_change
    ovs_poller.block()
  File "/usr/local/lib64/python3.10/site-packages/ovs/poller.py", line 231, in block
    events = self.poll.poll(self.timeout)
  File "/usr/local/lib64/python3.10/site-packages/ovs/poller.py", line 137, in poll
    rlist, wlist, xlist = select.select(self.rlist,
ValueError: filedescriptor out of range in select()
```

The way to avoid that is to replace the poller back to system-default select.poll like this:
```
from ovs import poller
import select
poller.SelectPoll = select.poll
```

This removes the limitation, since the poll() is correctly handling any number of file descriptors and not limited by their numerical values.  However, ovsdbapp is using multithreading, so I'm wondering what impact this change will have in runtime.
@otherwiseguy , maybe you can comment on this?

Reply:
I used your suggested method for fixing this by switching from `select` to `poll`